shopify/utils.py

```diff
@@ -72,3 +74,14 @@ def __validate_issuer_and_dest_match(cls, decoded_payload):
 
         if issuer_root != dest_root:
             raise MismatchedHostsError()
+
+
+def sanitize_shop_domain(shop_domain, myshopify_domain="myshopify.com"):
```

Comment:
we shouldn't make `myshopify_domain` configurable as it can allow anyone to override the shopify domain that they are on.

Reply:
It's configurable in the `shopify_app` gem today - I think it's to accommodate Plus shops (need to verify this)